source/crud/crud.rst

```diff
@@ -670,6 +685,12 @@ The OP_QUERY wire protocol only contains a numberToReturn value which drivers mu
 
 Because of this anomaly in the wire protocol, it is up to the driver to enforce the user-specified limit. Each driver MUST keep track of how many documents have been iterated and stop iterating once the limit has been reached. When the limit has been reached, if the cursor is still open, a driver MUST kill the cursor.
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Database-level aggregation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The server supports several collection-less aggregation source stages like ``$currentOp`` and ``$listLocalSessions``. The shell allows users to run ``db.aggregate``, which uses a value of 1 as the collection name and executes the operation against the entire database. Drivers support for database-level aggregation will allow users to receive a cursor from these collection-less aggregation source stages.
```

Comment:
Instead of defining this in terms of the shell, I would say that the database aggregate command requires a collection name of 1 for collection-less source stages.